Forms/main.css

```diff
@@ -1,57 +1,84 @@
 body{
-    width: 1800px;
-    height: 900px;
-    /*background-color: yellow;*/
+    width: 100vw;
+    height: 100vh;
 }
 
 div.container{
     display: flex;
-    margin: 10px    ;
+    margin: 10px;
+    justify-content: center;
+}
+div.head-container{
+    display: flex;
+    margin: 10px;
+    flex-direction: row-reverse;
+    justify-content: center;
 }
-
 h1{
     color: darkgray;
-    width: 95%;
+    width: 100%;
     height: 100px;
     text-align: center;
+    padding-right: 200px;
 }
 
 footer{
-    position: relative;
-    width: 95%;
-    height: 100px;
+    background-color: aquamarine;
+    width: 100% ;
+    height: 50px;
     text-align: center;
     margin: 40px 0 0 40px;
 }
 
 .img1 {
-    position: absolute;
+    position: relative;
+    display: flex;
     width: 200px;
     height: 150px;
-    margin: 10px;
-
+    margin-left: 40px;
 }
 
-div.first_collum{
-
+div.first_column{
+    width: 200px;
+    /*margin-left: 60px;*/
+    margin-right: 60px;
     display: flex;
-    justify-content: space-between;
-    flex-direction: column;
+    justify-content: center;
 }
 
-div.secound_collum{
-    width: 700px;
-    margin-left: 60px;
-    margin-right: 60px;
+div.second_column{
     display: flex;
     justify-content: center;
+    flex-direction: column;
+    flex: 2;
```

Comment:
What is `flex: 2` for? I'm just curious, maybe it's not incorrect

Reply:
Yes, i think first collum take more plase, after i found wight parameter in first column and change him. But flex: 2 is stay, i'll del him